### PR TITLE
polynomial inter-conversions

### PR DESCRIPTION
**Pull request by srajangarg**

### CODE REVIEW

File: symengine/polys/uintpoly_flint.h [hunk 1]
```diff
@@ -74,6 +74,17 @@ class UFlintPoly : public BaseType<Container, Poly>
         return make_rcp<const Poly>(var, std::move(f));
     }
 
+    template <typename FromPoly>
+    static RCP<const Poly> from_poly(const FromPoly &p)
```
Comment (isuruf) [inline symengine/polys/uintpoly_flint.h]: Can you enable this only if `FromPoly` is a univariate polynomial? Otherwise, it looks good.

**Comment by srajangarg** [comment]
@isuruf ping

**Comment by srajangarg** [comment]
@isuruf I will fix that another PR (there are some more places where this can be fixed). Will send one in soon, can you merge this in?

**Comment by certik** [comment]
@isuruf do you have objections to merging this PR as is and improving upon it later, or should it be fixed before we merge?

**Comment by isuruf** [comment]
@certik, go ahead. I forgot about this PR.

**Pull request merged**